### PR TITLE
Update to LibGit2Sharp 0.26.2

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

I update LibGit2Shart to 0.26.2

### CODE REVIEW

**Comment by soroshsabz** [comment]
@x-jokay why we got many errors in test.cake ?

**Comment by x-jokay** [comment]
@soroshsabz you can see the results of the builds [here](https://ci.appveyor.com/project/cakecontrib/cake-git/builds/30309483) and [here](https://travis-ci.org/cake-contrib/Cake_Git/builds/640905930).

This may be a problem due to [this](https://github.com/cake-contrib/Cake_Git/pull/115/files#diff-64308e660d481d0a88bcdcb115324f44L20) change.

**Comment by soroshsabz** [comment]
@x-jokay Thanks for replaying, I fixed this issue, and now all CI is green :)

File: src/Cake.Git/Cake.Git.csproj [hunk 1]
```diff
@@ -17,9 +17,9 @@
     <ProjectGuid>{08F0A5C3-0E9C-451D-B003-14FF73699BE1}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.33.0" PrivateAssets="All" />
-    <PackageReference Include="LibGit2Sharp" Version="0.26.0" />
-    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.267" />
+    <PackageReference Include="Cake.Core" Version="0.36.0" PrivateAssets="All" />
```
Comment (pascalberger) [inline src/Cake.Git/Cake.Git.csproj]: There's no reason to build against a newer version of Cake. Cake 0.33.0 is the earliest version with API compatibility and should be built against for maximum compatibility

File: tools/packages.config [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Cake" version="0.33.0" />
-    <package id="Cake.CoreCLR" version="0.33.0" />
+	<package id="Cake" version="0.36.0" />
```
Comment (pascalberger) [inline tools/packages.config]: Building with a newer version of Cake is unrelated to bumping LibGit2Sharp. Please remove it from this PR.

**Review by pascalberger — CHANGES_REQUESTED** [review]
@soroshsabz Thank you for the pull request. I added two comments which should be fixed before this can be merged.

**Comment by soroshsabz** [comment]
@pascalberger I downgrade Cake.Core , and I don't know why AppVeyor failed

**Comment by x-jokay** [comment]
@soroshsabz your last [commit](https://github.com/cake-contrib/Cake_Git/pull/115/commits/5d40fd2a44a8c62d91d708bb9007f746a48b7e85) is not necessary, just revert it and squash the remainig first 1-4 commits to one commit, so only the `version number` change in `Cake.Git.csproj` for `LibGit2Sharp` and `LibGit2Sharp.NativeBinaries` will remain in 1 commit.

**Comment by soroshsabz** [comment]
@x-jokay Ok I revert it and make one commit for change, please review it :)

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@soroshsabz your changes have been merged, thanks for your contribution 👍

**Pull request merged**